src/diffusers/models/modeling_utils.py

```diff
@@ -292,6 +295,149 @@ def disable_xformers_memory_efficient_attention(self):
         """
         self.set_use_memory_efficient_attention_xformers(False)
 
+    def add_adapter(self, adapter_config, adapter_name: Optional[str] = None) -> None:
```

Comment:
```suggestion
    def add_adapter(self, adapter_config, adapter_name: str = "default") -> None:
```

Wouldn't this work as well?

Reply:
Yes definitely!